app/src/test/java/com/codesoom/assignment/auth/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,148 @@
+package com.codesoom.assignment.auth.application;
+
+import com.codesoom.assignment.auth.infra.JwtTokenProvider;
+import com.codesoom.assignment.user.application.UserEmailNotFoundException;
+import com.codesoom.assignment.user.domain.User;
+import com.codesoom.assignment.user.domain.UserRepository;
+import io.jsonwebtoken.Claims;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class AuthenticationServiceTest {
+    private static final String SECRET = "12345678901234567890123456789010";
+    private static final String GIVEN_USER_EMAIL = "test@test.com";
+    private static final String GIVEN_USER_PASSWORD = "password";
+    private static final String NOT_EXIST_EMAIL = "not_exist@test.com";
+    private static final String WRONG_PASSWORD = "wrong_password";
```

Comment:
이렇게 하면 올바른 패스워드와 절대로 같지 않은 값이라는 걸 어느 정도 보장할 수 있습니다.
```suggestion
    private static final String WRONG_PASSWORD = GIVEN_USER_PASSWORD + "_WRONG";
```

Reply:
오 꿀팁감사합니다 ! 😄